spec/pivo_flow/cli_spec.rb

```diff
@@ -70,7 +70,7 @@
 
     it "and returns story id if file exists" do
```

Comment:
let's change to a newer syntax and make sure it works:

```ruby
it "and returns story id if file exists" do
  expect(File).to receive(:exists?).and_return(true)
  expect(File).to receive(:read).and_return("123")

  cli = PivoFlow::Cli.new
  expect(cli).to receive(:current_story_id_file_path).twice

  expect(cli.current_story_id).to eq "123"
end
```

Reply:
Yep sounds good.